src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -0,0 +1,72 @@
+package io.iohk.ethereum.domain
+
+import java.math.BigInteger
+
+import akka.util.ByteString
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.crypto.ECDSASignature
+import io.iohk.ethereum.rlp.{encode => rlpEncode, _}
+import io.iohk.ethereum.rlp.RLPImplicits._
+import org.spongycastle.util.encoders.Hex
+
+
+object SignedTransaction {
+
+  val FirstByteOfAddress = 12
+  val AddressLength = 20
+  val LastByteOfAddress = FirstByteOfAddress + AddressLength
+
+}
+
+case class SignedTransaction(tx: Transaction,
+                             pointSign: Byte, //v - 27 or 28 according to yellow paper, but it is 37 and 38 in ETH
+                             signatureRandom: ByteString, //r
+                             signature: ByteString /*s*/) {
```

Comment:
For multi-line declarations it's best to have each parameter on it's own line, then we don't need to re-indent if the name changes: http://docs.scala-lang.org/style/declarations#classes